### DISCUSSION TITLE
Queries hang

### DISCUSSION THREAD

**Discussion by harveyliu168** [milvus-io/milvus#19154] [Q&A and General discussion]
We have deployed miluvs on GKE. From past couple of days, the queries just hang with some of collections.  There are no error. If you check the logs, the queries have started ok, but never complete. Could you someone please help me out? I am really stuck here.

**Comment by yhmo** [comment]
This kind of problem is not easy to investigate, it is better to create an issue on the [Issues](https://github.com/milvus-io/milvus/issues) panel. Please provide more information including version/log in the issue and we will assign to the proper engineer.
